pcdsdevices/lasers/thorlabsWFS.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from ophyd.device import Component as Cpt
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+from pcdsdevices.areadetector.detectors import PCDSAreaDetectorTyphos
+
+logger = logging.getLogger(__name__)
+
+
+class ThorlabsWfs40(PCDSAreaDetectorTyphos):
+    """Class to implement a Thorlabs WFS40 Wavefront sensor."""
+
+    beam_status = Cpt(EpicsSignalRO, 'StatHighAmbientLight', kind='normal')
+
+    pupil_centroid_x = Cpt(EpicsSignal, 'PupilCentroidX', kind='config')
+    pupil_centroid_y = Cpt(EpicsSignal, 'PupilCentroidY', kind='config')
+    pupil_diameter_x = Cpt(EpicsSignal, 'PupilDiameterX', kind='config')
+    pupil_diameter_y = Cpt(EpicsSignal, 'PupilDiameterY', kind='config')
+
+    pupil_centroid_x_rbv = Cpt(EpicsSignal, 'PupilCentroidX_RBV',
+                               kind='normal')
+    pupil_centroid_y_rbv = Cpt(EpicsSignal, 'PupilCentroidY_RBV',
+                               kind='normal')
+    pupil_diameter_x_rbb = Cpt(EpicsSignal, 'PupilDiameterX_RBV',
+                               kind='normal')
+    pupil_diameter_y_rbv = Cpt(EpicsSignal, 'PupilDiameterY_RBV',
+                               kind='normal')
```

Comment:
Are these RBV PVs are meant to be `EpicsSignalRO`?

Reply:
No, that would be giving me too much credit. I wrote most of this prior to learning about that signal type, and I forgot to update it here.

Reply:
Yes, they should be RO.